### PR TITLE
Fix typos in API specification

### PR DESCRIPTION
**Pull request by bocytko**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/134" target="_blank">ReviewNinja

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -421,16 +421,16 @@ paths:
...
         chunk (i.e. last delivered Event). The cursor might specify the offset with the symbolic value
         `OLDEST`, which will open the stream starting from the oldest available offset in the partition.
         Currently this format is the only one supported by the system, but in the future other MIME types
-        will be suported (like `application/event-stream`).
+        will be supported (like `application/event-stream`).
 
         If streaming for several distinct partitions, each one is an independent `EventStreamBatch`.
 
-        The initialization of a stream can be parameterized in terms of size of each chunk, timeout for
+        The initialization of a stream can be parametrized in terms of size of each chunk, timeout for
```
Comment (tmuehl) [inline api/nakadi-event-bus-api.yaml]: I think "parameterized" is actually ok:
https://dict.leo.org/ende/?lang=de&search=parameterized

**Comment by tmuehl** [comment]
@bocytko Thanks for fixing! :)

File: api/nakadi-event-bus-api.yaml (hunk 1 above)
Comment (bocytko) [inline api/nakadi-event-bus-api.yaml]: Thanks for noticing that. It's corrected.

**Comment by bocytko** [comment]
@tmuehl you're welcome! :smile:

**Pull request merged**